### DISCUSSION TITLE
Monospaced font packing

### DISCUSSION THREAD

**Discussion by CCDD564** [nothings/stb#1242] [Q&A] [answered]
Is there a way to tell stb truetype to pack glyphs in equally sized cells instead of packing them closely together, when using monospaced fonts?  
Example:
![Capture](https://user-images.githubusercontent.com/24461970/140065558-2dae2040-275d-43a7-856f-61fbf6cabfc5.PNG)

**Comment by nothings** [comment, marked as answer]
No, the "bake a bunch of glyphs" API doesn't support this, but it's very easy to do yourself. Normally packing is complicated, which is why there's a routine to do it for you, but for this case you can just iterate through the grid and render each one, one at a time.
